RA_Integration/RA_Dlg_Memory.cpp

```diff
@@ -939,11 +951,20 @@ INT_PTR Dlg_Memory::MemoryProc( HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPar
 				else //if( b32BitSet )
 					nCompSize = ThirtyTwoBit;
 
-				g_MemManager.ResetAll( nCompSize );
+				ByteAddress start, end;
+				if( GetSelectedMemoryRange( start, end ) )
+				{
+					g_MemManager.ResetAll( nCompSize, start, end );
 
-				ClearLogOutput();
-				AddLogLine( "Cleared: (" + std::string( COMPARISONVARIABLESIZE_STR[ nCompSize ] ) + ") mode. Aware of " + std::to_string( g_MemManager.NumCandidates() ) + " RAM locations." );
-				EnableWindow( GetDlgItem( hDlg, IDC_RA_DOTEST ), g_MemManager.NumCandidates() > 0 );
+					ClearLogOutput();
+					AddLogLine( "Cleared: (" + std::string( COMPARISONVARIABLESIZE_STR[ nCompSize ] ) + ") mode. Aware of " + std::to_string( g_MemManager.NumCandidates() ) + " RAM locations." );
+					EnableWindow( GetDlgItem( hDlg, IDC_RA_DOTEST ), g_MemManager.NumCandidates() > 0 );
+				}
+				else
+				{
+					ClearLogOutput();
+					AddLogLine( "Invalid Range" );
+				}
```

Comment:
This block calls MemManager::ResetAll with the specified range, or outputs an Invalid Range error if the custom range cannot be parsed.